cfgov/wagtailadmin_overrides/templates/wagtailadmin/login.html

```diff
@@ -1,79 +1,7 @@
 {% extends "wagtailadmin/login.html" %}
 {% load i18n wagtailadmin_tags %}
```

Comment:
This line is no longer needed as the template now only contains a simple HTML block.

```suggestion
```